tests/unit_tests/scheduler/test_openpbs_driver.py

```diff
@@ -106,3 +107,69 @@ async def test_job_name_with_prefix():
     driver = OpenPBSDriver(job_prefix="pre_")
     await driver.submit(0, "sleep", name="sleepy")
     assert " -Npre_sleepy " in Path("captured_qsub_args").read_text(encoding="utf-8")
+
+
+@pytest.mark.parametrize(
+    ("exit_code, error_msg"),
+    [
+        (171, "Invalid credential"),
```

Comment:
Maybe we can replace this with Enum stuff:

```
import enum
class ExitCodes(enum.Enum):
    INVALID_CREDENTIAL =  171
```
```
then:
>>> ExitCodes.INVALID_CREDENTIAL.name.lower().capitalize()
'Invalid_credential'
```

Reply:
I would like to have it more explicit, so I use the full message from the stderr.